_posts/2016-02-29-phishing.md

```diff
@@ -0,0 +1,54 @@
+---
+layout:     post
+title:      Preventing phishing attacks - an example of defense in depth
+author:     dan_adams
+date:       2016-02-29 12:00:00
+summary:    Phishing attacks are difficult to entirely prevent against, but are a good example of a situation in which "defense in depth" is appropriate.
+category:   security
+tags:       security
+---
+
+Preventing phishing attacks - an example of defense in depth
+
+"Phishing" attacks are attempts to acquire information such as usernames, passwords, and credit card details by masquerading as a trustworthy entity. The name derives from the similarity of using a bait in an attempt to catch a victim from a pool and the "bait" normally consist of an electronic message such as an email sent to a large group with the hope that some small percentage of targeted victims will fall for the ploy.
+
+A phishing email is generally sent by a malicious source but purporting to be from a known or trusted source such as a company's IT department or a popular social media website or high street bank. Phishing emails will often either have a malicious payload such as a word document with a macro, or else contain links to websites that are either infected with malware or attempt to trick people into entering their credentials.
+
+Typically the email itself, or the website linked to from the email use corporate imagery, branding and fonts to attempt to provide a look and feel are almost identical to the genuine, legitimate business being impersonated.
+
+
+Phishing attacks are difficult to entirely prevent against, but are a good example of a situation in which "defense in depth" is appropriate - no one measure on its own is effective, but by implementing a range of anti-phishing countermeasures, the effectiveness of phishing attacks can be greatly reduced. So what can be done?
+
+### 1. Server-side Email filtering
+
+The first line of defense needed is anti-spam and email filtering for inbound email, to prevent emails from being received by your employees in the first place. Various technologies can be used including anti-spam filter based on BLs and IP reputation services, a server-side antivirus scan for attachments, and MX record/domain verification using Sender Policy Framework (SPF). Emails can be rejected by your mailserver so that they never make it to your users.
```

Comment:
Either "an anti-spam filter..." or "including anti-spam filters...".

Also, maybe "Emails can then be rejected by..."